net/redisclient.go

```diff
@@ -226,11 +248,43 @@ func NewRedisRingClient(ro *RedisOptions) *RedisRingClient {
 		r.ring = redis.NewRing(ringOptions)
 		r.log = ro.Log
 		r.metricsPrefix = ro.MetricsPrefix
+
+		if ro.AddrUpdater != nil {
+			if ro.UpdateInterval == 0 {
+				ro.UpdateInterval = defaultUpdateInterval
+			}
+			go func() {
+				old := len(ringOptions.Addrs)
+				t := time.NewTicker(ro.UpdateInterval)
+				defer t.Stop()
+				r.log.Info("Start goroutine to update redis instances every %s", ro.UpdateInterval)
+				for range t.C {
```

Comment:
I think this should `for { }` with a select from  t.C and r.quit, otherwise quit have to wait for another ticker cycle

Reply:
yes true, done